patches/server/0353-Anti-Xray.patch

```diff
@@ -845,9 +845,7 @@ index 0000000000000000000000000000000000000000..a5d43901595f864c8a5dd5d013aa42d7
 +
 +        HIDE(1, "hide ores"),
 +        OBFUSCATE(2, "obfuscate");
-+
```

Comment:
personally I think the whitespace here improves readability

Reply:
Moved to config patch and done.